.github/workflows/build.yml

```diff
@@ -7,8 +7,6 @@ name: CI
 on:
   push:
     branches: [ master ]
-  pull_request:
-    branches: [ master ]
```

Comment:
This appears like it would cause deploy even when PRs are made to master. This doesn't seem correct so I removed it.